### PR TITLE
rfac: use sloth videos

### PR DESCRIPTION
**Pull request by zhixiangteoh**
Stereo top-bottom (360 deg): https://d25a56pc18k0co.cloudfront.net/sloths_binaural_3840x2160_360_3D_v2_injected.mp4
Stereo-left-right (180 deg): https://d25a56pc18k0co.cloudfront.net/sloths_binaural_3840_180_3D-injected.mp4

Added these necessary lines to `createVideo` function:
```js
video.crossOrigin = "anonymous";
video.preload = "auto";
video.autoload = true;
```

### CODE REVIEW

File: apps/simple-video-layers/multiple-layers.js [hunk 1]
```diff
@@ -2,14 +2,18 @@ import * as THREE from "three";
...
 import { XRControllerModelFactory } from "three/examples/jsm/webxr/XRControllerModelFactory";
 
-import panoVideo from "../../media/pano.mp4";
 import buttonClickSound from "../../media/audio/button-click.mp3";
 import MediaLayerManager from "../../util/MediaLayerManager";
 import { WebGLRenderer } from "../../util/WebGLRenderer";
 import { VRButton } from "../../util/webxr/VRButton";
 
+const SLOTH_TOP_BOTTOM_VIDEO =
+    "https://d25a56pc18k0co.cloudfront.net/sloths_binaural_3840x2160_360_3D_v2_injected.mp4";
```
Comment (und3fined-v01d) [inline apps/simple-video-layers/multiple-layers.js:11]: lets download this instead of a url

**Review by und3fined-v01d — CHANGES_REQUESTED** [review]

File: apps/simple-video-layers/multiple-layers.js [hunk 2]
```diff
@@ -286,6 +293,9 @@ class App {
     createVideo(videoIn) {
         const video = document.createElement("video");
         video.loop = true;
+        video.crossOrigin = "anonymous";
+        video.preload = "auto";
```
Comment (und3fined-v01d) [inline apps/simple-video-layers/multiple-layers.js:297]: don't need this if we download the videos

**Review by und3fined-v01d — CHANGES_REQUESTED** [review]

File: apps/simple-video-layers/multiple-layers.js (hunk 1 above)
Comment (zhixiangteoh) [reply apps/simple-video-layers/multiple-layers.js:11]: Don't download it!!! Trust me I tried and it's too big... even spent some time lowering quality to 1080p and trimming video but it's still too big to load.

File: apps/simple-video-layers/multiple-layers.js (hunk 2 above)
Comment (zhixiangteoh) [reply apps/simple-video-layers/multiple-layers.js:297]: See above.

File: apps/simple-video-layers/multiple-layers.js (hunk 1 above)
Comment (cabanier) [reply apps/simple-video-layers/multiple-layers.js:11]: Yes, these files are very big so you can't host them on github.
Also, if you downsample them, they will no longer look good.

**Comment by cabanier** [comment]
I think this can be merged once the conflicts are resolved?

**Pull request merged**